src/components/clientDashboard/enrollments/AllEnrollments.tsx

```diff
@@ -92,6 +92,8 @@ const AllEnrollments = () => {
           columns={columns}
           pagePath='client.enrollments'
           fetchPolicy='cache-and-network'
+          showFilters
+          recordType='Enrollment'
         />
```

Comment:
This isn't looking quite how I would expect:
1) There are no filters
2) It doesn't make sense to sort on "household id" here since the household ID is not shown (because it is different per row in this context). So, we should hide the sort functionality in this view.
![Uploading Screen Shot 2023-05-31 at 1.25.34 PM.png…]()

Reply:
Can you enable sorting and filtering on this table (ProjectClientEnrollmentsTable, I believe)? It is mentioned in the ticket. The ticket also says to update the household table, which you can skip I suppose, since that will require updating the sort and filter options for the household query.


<img width="1337" alt="Screen Shot 2023-05-31 at 1 29 16 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/8f2ad443-a30f-4125-ab72-d40e969bb58d">